app/full_text_search.py

```diff
@@ -170,21 +215,35 @@ def create_document(record_id, repo, **kwargs):
     fields.append(appengine_search.TextField(name='repo', value=repo))
     fields.append(appengine_search.TextField(name='record_id', value=record_id))
 
-    # Add name and location romanized by unidecode
-    for field in kwargs:
-        romanized_value = script_variant.romanize_word(kwargs[field])
-        fields.append(
-            appengine_search.TextField(name=field, value=romanized_value))
+    # Add name romanized by unidecode
+    fields.extend(create_romanized_name_fields(
```

Comment:
You have two similar calls of create_romanized_name_fields and create_romanized_location_fields. Can you simplify the code here using for-loop?